phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionQueryServicesImpl.java

```diff
@@ -1066,6 +1067,13 @@ private void addCoprocessors(byte[] tableName, HTableDescriptor descriptor, PTab
                     }
                 }
             }
+
+            if (!SchemaUtil.isSystemTable(tableName)) {
+                if (!descriptor.hasCoprocessor(TTLAwareRegionObserver.class.getName())) {
+                    descriptor.addCoprocessor(TTLAwareRegionObserver.class.getName(), null, priority-2, null);
```

Comment:
Can you please add a comment to mention why this Co-processor priority has to be so high?